website/docs/r/policy_ip_discovery_profile.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+subcategory: "Policy - Segments"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_ip_discovery_profile"
+description: A resource to configure an IP discovery profile.
+---
+
+# nsxt_policy_ip_discovery_profile
+
+This resource provides a method for the management of IP discovery profiles.
+
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_ip_discovery_profile" "ip_discovery_profile" {
+  description  = "ip discovery profile provisioned by Terraform"
+  display_name = "ip_discovery_profile1"
+
+  arp_nd_binding_timeout         = "20"
```

Comment:
please remove quotes around integers and booleans, only strings should be quoted